config/routes.rb

```diff
@@ -62,6 +62,8 @@
 
         resources :news_items, only: %i[index show create update destroy],
                                controller: 'news/items'
+
+        resources :category_assessments, module: 'green_lanes', only: %i[index]
```

Comment:
We shouldn't be exposing green lanes admin API for uk only? If anything I think it should be XI only sicnce that is where the data is